.github/workflows/release.yaml

```diff
@@ -136,9 +136,54 @@ jobs:
         run: |
           RELEASE_VERSION=${{ needs.publish.outputs.release_version }}
           TAG_COMMIT=$(git rev-list -n 1 "$RELEASE_VERSION")
-          BASE_BRANCH=$(git branch -r --contains "$TAG_COMMIT" | grep -v HEAD | sed -e 's/^[[:space:]]*//' -e 's/^origin\///' | head -n 1 || echo "main")
```

Comment:
I'm confused why all this distance stuff is necessary, can't you just pull the version from the tag like above ? `startsWith(github.ref, 'refs/tags/v') == true`